src/components/AntiPatternActionsComponent.vue

```diff
@@ -1,19 +1,19 @@
 <template>
     <span>
         <v-tooltip top>
-            <v-btn slot="activator" icon>
+            <v-btn slot="activator" icon v-clipboard:copy="antiPatternAsJson">
                 <v-icon>content_copy</v-icon>
             </v-btn>
             <span>Copy the json file</span>
         </v-tooltip>
         <v-tooltip top>
-            <v-btn slot="activator" icon>
+            <v-btn slot="activator" icon v-clipboard:copy="antiPattern.sources[0]">
```

Comment:
Why only copy the first BibTex entry?

Reply:
should  i copy both or all ? and all than as JSON?